app/controllers/surveys_controller.rb

```diff
@@ -1,5 +1,8 @@
 class SurveysController < ApplicationController
   
+  before_action do
+    @groups = Group.all
```

Comment:
do you need groups in every action 
can we do the same thing using single query ?